changelog.d/4927.feature

```diff
@@ -0,0 +1 @@
+Batch up outgoing read-receipts to reduce federation traffic.
```

Comment:
Looks like the wrong description for this PR?

Reply:
nope, it's deliberately the same as the newsfile for the PR which introduced the bug (#4890, ftr), so that they get merged into one entry in the changelog